### ISSUE TITLE
Deploy scancode.io through helm chart

### ISSUE THREAD

**Issue by shusriva** [nexB/scancode.io#461]
Hi Team,
Can you please provide a helmchart to deploy scancode.io in Kubernetes cluster.
Currently creating and managing helm chart for scancode.io is a pain

**Comment by AyanSinhaMahapatra** [comment]
See https://github.com/xerrni/scancode-kube (and let us know how this goes). We have this docs in a PR #442 which we have to merge in soon.

**Comment by shusriva** [comment]
@AyanSinhaMahapatra one of the pvc is not getting bound because of which pods are not coming up
![image](https://user-images.githubusercontent.com/35764437/177474847-e3b1b4dc-f446-401a-809d-61e70b7c87a5.png)

**Comment by tdruez** [comment]
The documentation is now available at https://scancodeio.readthedocs.io/en/latest/installation.html#helm-chart-beta

> Somehow the UI is not looking as expected.

@shusriva This looks like an issue with the static files (CSS, JS, ...) that are not properly setup.

@xerrni any ideas/suggestions about this issue?

**Comment by shusriva** [comment]
@xerrni @AyanSinhaMahapatra 
After debugging I found that somehow  scancode-staticsite pvc is not getting bound to any pod.
![image](https://user-images.githubusercontent.com/35764437/177723030-dbac4435-fd4a-47a6-8b4d-3e5e52f1f941.png)

**Comment by tdruez** [comment]
@shusriva were you able to fix the `staticsite` issue? I'd like to improve the current documentation if you have some feedback on this.

**Comment by shusriva** [comment]
@tdruez Yes, values.yaml file was missing under scancode folder. I just placed it there. 
Somehow I am able to make it up and running but the helm chart(scancode) is very tightly coupled with other helm charts (postgres,nginx and redis). some images and hostname are hardcoded in the deployment file.

As per my requirement, I have to utilize existing Postgres and Redis from the cluster but I wasn't able to do this with this helm chart.

**Comment by tdruez** [comment]
Feedback on the current implementation from @shusriva at https://github.com/nexB/scancode.io/issues/541#issuecomment-1271418139

> 1. Chart need some more improvement, I was unable to configure my existing postgres and redis because of dynamic cred. env variables needs to be configured in the chart to use existing psotgres and redis

> 2. storage class is not defined explicitly in values.yaml file, hence we are forced to use local path for scancode. please provide options in values.yaml to explictly mention the storage class

> 3. scancode pods are not coming up if any other storage class is used (e.g ebs-ssd from aws)
